### DISCUSSION TITLE
Welcome to crow_frontend Discussions!

### DISCUSSION THREAD

**Discussion by picoral** [writecrow/crow_frontend#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OlaSwa** [comment]
I've noticed we have a certain inconsistency visually signalling a fragment of a text in the corpus is a "corpus text". In our results page 
![image](https://user-images.githubusercontent.com/77394887/152830798-b53ffdf1-ccf7-4280-944c-5a38630f134b.png) the texts are on a white background. 

When we click "View text", the corpus text has the same white background as the "informational / metadata" commentary above it. I think in this preview section, I would like to see the corpus text be visually separated from the interface text. I get this feeling where I don't know where the sample starts and ends because of the colors on this page. 

![image](https://user-images.githubusercontent.com/77394887/152831271-7243ce8d-81fa-4b63-acf9-d2312e0ac41b.png)

Update: I wonder how we would also justify showing samples a certain way in the embedding and results page with style:
Fonts: Verdana,Geneva,Tahoma,sans-serif; 

versus on the preview with: 
- raleway, sans-serif. 
 
I think staying consistent with the fonts would also visually signal corpus texts.

**Comment by markfullmer** [comment]
Hi @OlaSwa , I think this is a very sensible UX improvement. Here's a mock-up of what that change would look like. 

![Screenshot from 2022-02-27 10-19-09](https://user-images.githubusercontent.com/4305692/155890401-27eb670c-23a9-4c13-8482-2abfa8c28ba3.png)

I'd be inclined to consider the font change separately. We use a monospace font in the embedded results because those show multiple results, and we want the spacing between the letters to be consistent, so that the text between excerpts "lines" up. In other words, the font choice here is not so much meant to indicate "this is a text" as to accomplish a positioning requirement when viewing multiple excerpts.

That said, if we feel like it is useful to help indicate "this is a text" in the web interface, I'm not opposed to that.
